app/src/pages/inside/common/dashboardPageHeader/dashboardPageHeader.jsx

```diff
@@ -33,6 +33,9 @@ import {
 } from 'controllers/dashboard';
 import { InputDropdown } from 'components/inputs/inputDropdown';
 import { NavLink } from 'components/main/navLink';
+import { userRolesSelector } from 'controllers/user';
+import { userRolesType } from 'common/constants/projectRoles';
+import { canWorkWithDefectTypes } from 'common/utils/permissions/permissions';
```

Comment:
I'm not sure it is a right permission here.
I suppose it should be connected with dashboards.